admin/dw-options.php

```diff
@@ -58,6 +58,22 @@ function dw_render_options_page() {
 
 			<h2 class="title">Standaard waarden voor nieuwe cursussen</h2>
 			<table class="form-table">
+				<tr>
+					<th scope="row">
+						<label class="description" for="dw_settings[ccvCode]">
+							<?php _e('Training code'); ?>
+						</label>
+					</th>
+					<td>
+						<input id="dw_settings[ccvCode]" name="dw_settings[ccvCode]"
+							   type="text"
+							   value="<?php echo $dw_options['ccvCode'] ?>"
+						>
+						<p class="description">
+							Trainingcode om op te filteren, gescheiden door een <code>;</code>. Leeg laten als alle trainingen gesynchroniseerd moeten worden
```

Comment:
Maybe something like:

> Trainingscodes gescheiden door een `;`. Alleen trainingen waarvan de trainingscode in de lijst staat worden gesynchroniseerd. Indien het veld leeg is, worden alle trainingen gesynchroniseerd.  


I missed the part that said whether this is a blacklist or whitelist.